app/shownoter.py

```diff
@@ -158,29 +155,26 @@ def valid_link(site):
 class Link():
     """Class that creates the link dictionary reference holding the site information"""
 
-    def __init__(self, is_image=False):
```

Comment:
lol.... still not sure why this isn't in an __init__ call.

especially since it is false by default. 

It seems somewhat arbitrary but perhaps you can shed some light on the subject. (Possibly again)